DropDownMenu.swift

```diff
@@ -117,7 +122,18 @@ open class DropDownMenu : UIView, UITableViewDataSource, UITableViewDelegate, UI
 		super.layoutSubviews()
         let contentHeight = menuCells.reduce(0) { $0 + $1.rowHeight }
 		
-		menuView.frame.size.height = contentHeight
+        //  Set the right height for tableView
```

Comment:
Would be be nice to indent it with tabs to match the rest of the code. 

Take note that the `let contentHeight = ` line is incorrectly indented. This file has a few indentation issues, I'll fix them in another commit.